ons_alpha/images/models.py

```diff
@@ -1,7 +1,24 @@
+from typing import NamedTuple
+
+from django.core.files.base import ContentFile
 from django.db import models
 from wagtail.images.models import AbstractImage, AbstractRendition, Image
 
 
+class StubImage(NamedTuple):
+    """
+    A stub image class for use when generating rendition image URLs.
+    """
+
+    id: int
+
+    @property
+    def file(self):
+        # HACK: Provide a blank filename to satisfy `reverse`,
+        # but the view never uses it.
+        return ContentFile("", name="")
```

Comment:
What if we did want to generate actual rendition URLs?. I think this needs a bit of clarity on why we are doing this.
What the scenarios that currently fail without this, and what may the actual solutions look like?.